aibolit/patterns/empty_rethrow/empty_rethrow.py

```diff
@@ -19,29 +19,28 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
-import javalang
-
+from aibolit.ast_framework import ASTNodeType, AST
 from aibolit.utils.ast_builder import build_ast
+from typing import List, Set
 
 
 class EmptyRethrow:
+    '''
+    Check if we throw the same exception as it was caught
+    '''
 
-    def __init__(self):
-        pass
+    def value(self, filename) -> List[int]:
+        total_code_lines: Set[int] = set()
+        ast = AST.build_from_javalang(build_ast(filename))
+        for try_node in ast.get_proxy_nodes(ASTNodeType.TRY_STATEMENT):
+            for throw_node in ast.get_proxy_nodes(ASTNodeType.THROW_STATEMENT):
+                field_catche = try_node.catches
+                if field_catche:
+                    catch_classes = [x.parameter.name for x in field_catche]
+                    throw_child = list(throw_node.children)[0]
+                    if throw_child.node_type == ASTNodeType.CLASS_CREATOR:
+                        continue
+                    if hasattr(throw_child, 'member') and throw_child.member in catch_classes:
```

Comment:
- This two `if`s are totally about one thing. Should we add line or not. Merge them into a single `if`.
 - According to `hasattr`, I believe, that it must be there, but it might be `None`, so lets check it for `None`
 - The check with `CLASS_CREATOR` need some comment

Reply:
1) rewritten
2) not agree with you, it doesn't work like that. it throw exception that this node doesnt have this attribute
3) deleted it. it was extra checking